WTWSMS/common/buildings/WtWSMS_historical_temple.txt

```diff
@@ -0,0 +1,183 @@
+temple = {
+
+	tp_hagiasophia = {  # Hagia Sophia
+		potential = {
+			FROMFROM = { title = b_hagiasophia }
+			FROM = { had_global_flag = { flag = hagia_sophia_built days = 0 } }
+		}
```

Comment:
It doesn't, but the important matter is that it is impossible to build it directly.